libs/core/langchain_core/prompts/base.py

```diff
@@ -70,10 +70,24 @@ def get_input_schema(
     def invoke(
         self, input: Dict, config: Optional[RunnableConfig] = None
     ) -> PromptValue:
+        def format_prompt(inner_input: Dict) -> PromptValue:
```

Comment:
nit: does it need to be nested function?

Reply:
Could do outside